contribs/src/test/java/com/netflix/conductor/contribs/lock/LocalOnlyLockTest.java

```diff
@@ -12,107 +12,84 @@
  */
 package com.netflix.conductor.contribs.lock;
 
-import org.junit.Ignore;
+import java.util.concurrent.TimeUnit;
+
+import org.junit.Test;
+import org.springframework.boot.test.context.runner.ApplicationContextRunner;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
 
-@Ignore
-// Test causes "OutOfMemoryError: GC overhead limit reached" error during build
 public class LocalOnlyLockTest {
 
-    //    // Lock can be global since it uses global cache internally
```

Comment:
Since you are on to fix the LocalOnlyLock, could you also take a look whether we can run those tests instead of comment out?

Reply:
Hi @jxu-nflx , Please see below they are uncommented. Only one test I have removed because that can cause the memory issue.

Reply:
Hi @jxu-nflx , Please clarify on this.

Reply:
I see, never mind, I saw those tests are added back.